.github/workflows/skipped-integration-tests.yml

```diff
@@ -0,0 +1,21 @@
+# This is a workaround due to Github Actions limitations on aggregating results from a job matrix.
+
+name: Skipped Integration Tests
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        # These are all the Python versions that we support.
+        python-version: ['3.7', '3.8', '3.9', '3.10']
```

Comment:
why is the matrix necessary if we run `exit 0` here?

Reply:
Rather than enumerate all the job names by hand, I decided to use the same matrix as the one in `integration-test.yml` to make it easier to reproduce names. This is just a more convenient way to generate the needed job names.

Reply:
The `exit 0` is to make sure all these dummy jobs pass.